Sources/PkgConfig/PkgConfig.swift

```diff
@@ -10,12 +10,18 @@
 
 import Utility
 import func POSIX.getenv
+import func POSIX.popen
 
 enum PkgConfigError: ErrorProtocol {
     case CouldNotFindConfigFile
     case ParsingError(String)
 }
 
+private let pkgConfigSearchPaths: [String] = {
+    let searchPaths = try? POSIX.popen(["pkg-config", "--variable", "pc_path", "pkg-config"])
```

Comment:
we should probably check if pkg-config is installed first or does it return nil/empty strings if this errors out?